lgc/patch/RegisterMetadataBuilder.cpp

```diff
@@ -162,8 +162,10 @@ void RegisterMetadataBuilder::buildLsHsRegisters() {
   // Minimum and maximum tessellation factors supported by the hardware.
   constexpr float minTessFactor = 1.0f;
   constexpr float maxTessFactor = 64.0f;
-  getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::VgtHosMinTessLevel] = FloatToBits(minTessFactor);
-  getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::VgtHosMaxTessLevel] = FloatToBits(maxTessFactor);
+  getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::VgtHosMinTessLevel] =
+      llvm::bit_cast<uint32_t>(minTessFactor);
+  getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::VgtHosMaxTessLevel] =
+      llvm::bit_cast<uint32_t>(maxTessFactor);
```

Comment:
The `llvm::` prefix should not be needed (I see that upstream llvm is not really consistent about this for `bit_cast`).

Reply:
Yes, true. I think I've missed some cases as well...